docs/complexity.md

```diff
@@ -51,6 +51,15 @@ In next step, you need to integrate `graphql-query-complexity` with your graphql
         onComplete: (complexity: number) => {
           console.log("Query Complexity:", complexity);
         },
+        estimators: [
+          fieldConfigEstimator(),
+          // Add more estimators here...
+          // This will assign each field a complexity of 1 if no other estimator
+          // returned a value.
+          simpleEstimator({
```

Comment:
And that here we can define the default value for field not explicitly annotated.

Reply:
done